release-notes/changes-in-timescaledb-2.md

```diff
@@ -320,6 +320,45 @@ After upgrading to TimescaleDB 2.0, **retention policies will no longer fail due
 continuous aggregates** and users have to ensure that retention and continuous aggregate policies have the 
 desired interplay. 
 
+Another change in 2.0 is that `drop_chunks` and the retention policy will no longer 
+automatically refresh continuous aggregates to account for changes in raw data 
+after the last refresh. Previously, the goal was to ensure that all updates were processed 
+prior dropping. However, it often didn't work as intended.
+
+In TimescaleDB 2.0 users can ensure that all updates are processed before dropping data by 
+combining a following experimental function, which refreshes a chunk, with drop_chunks.
```

Comment:
```suggestion
combining the following experimental function, which refreshes a chunk, with `drop_chunks`.
```

Also, I am always a bit hesitant when we say "refresh a chunk", since we are not really refreshing the chunk itself. We are refreshing the cagg based on data in the chunk.

Maybe this would work (just to avoid that issue):

```suggestion
combining the following experimental chunk-based refresh function with `drop_chunks`.
```

Reply:
> Also, I am always a bit hesitant when we say "refresh a chunk", since we are not really refreshing the chunk itself. We are refreshing the cagg based on data in the chunk.

I was also hesitant to say not fully correctly. On the other side it is not easy to be correct and short. I made an improvement there.